webapp/components/survey/CategorySelector/CategorySelector.js

```diff
@@ -73,7 +62,7 @@ const CategorySelector = (props) => {
           {i18n.t('common.manage')}
         </button>
       )}
-      {showCategoriesPanel && !showCategoryPanel && (
+      {showCategoriesPanel && (
```

Comment:
does the category list update the list after closing the category panel ?

Reply:
yes!